e2e/poms/formBuilder.ts

```diff
@@ -0,0 +1,124 @@
+import { Page, expect } from "@playwright/test";
+import { FORMBUILDER_SELECTORS } from "../constants/selectors";
+import { FORMBUILDER_API } from "../constants/selectors/formBuilder";
+
+export default class FormBuilder {
+    constructor(private page: Page) {
+        this.page = page;
+    }
+
+    setFormName = async (formName: string) => {
+        await this.page.getByTestId(FORMBUILDER_SELECTORS.formTitle).click();
+        await this.page.getByTestId(FORMBUILDER_SELECTORS.formRenameTextField).fill(formName);
+        await this.page.getByTestId(FORMBUILDER_SELECTORS.formRenameSubmitButton).click();
+
+        await expect(this.page.getByTestId(FORMBUILDER_SELECTORS.formTitle)).toContainText(formName);
+    };
+
+    addElement = async (elementName: string) => {
+        await this.page.getByRole('button', { name: elementName }).click();
+        await expect(this.page.getByTestId(FORMBUILDER_SELECTORS.questionHeader)).toContainText(elementName);
+    };
+
+    navigateToSubmissionsTab = async () => {
+        await this.page.getByTestId(FORMBUILDER_SELECTORS.submissionsTab).click();
+        await expect(this.page.getByTestId(FORMBUILDER_SELECTORS.submissionsCountLabel)).toBeVisible();
+    };
+
+    navigateToAnalytics = async () => {
+        await this.page.getByTestId(FORMBUILDER_SELECTORS.moreDropdownIcon).click();
+        await this.page.getByTestId(FORMBUILDER_SELECTORS.analyticsMoreTab).click();
+        await expect(this.page.getByTestId(FORMBUILDER_SELECTORS.submissionsInsightsTitle)).toBeVisible();
+    }
+
+    publishForm = async () => await this.page.getByTestId(FORMBUILDER_SELECTORS.publishButton).click();
```

Comment:
Avoid redundant awaits:
```suggestion
    publishForm = this.page.getByTestId(FORMBUILDER_SELECTORS.publishButton).click;
```